app/jobs/close_subjects_job.rb

```diff
@@ -23,8 +23,8 @@ def perform(*_args)
 
     # Close subjects who are past the monitoring period (and are actually closable from above logic)
     eligible.find_each do |subject|
-      if (!subject.last_date_of_exposure.nil? && subject.last_date_of_exposure <= (ADMIN_OPTIONS['monitoring_period_days'] + 1).days.ago) ||
-         (subject.last_date_of_exposure.nil? && subject.created_at < (ADMIN_OPTIONS['monitoring_period_days'] + 1).days.ago)
+      if (!subject.last_date_of_exposure.nil? && subject.last_date_of_exposure <= (ADMIN_OPTIONS['monitoring_period_days']).days.ago) ||
+         (subject.last_date_of_exposure.nil? && subject.created_at < (ADMIN_OPTIONS['monitoring_period_days']).days.ago)
```

Comment:
I agree with removing the +1, I think right now it's one day too later than it should be. Why do we only do < for the `created_at` check rather than <=?

Reply:
Good catch, but that is sort of an edge case anyways (LDE is required to exist, and does in most cases). Let me tweak it anyways.